innovation.js

```diff
@@ -1282,7 +1282,7 @@ function (dojo, declare) {
             
             dojo.place(div_title, dojo.body());
             
-            // Determine the font-size between 1 and 30px which enables to fill the container without overflow
```

Comment:
I updated things to use rem, so sizes are just 1-30 now, no need to specify unit.

Reply:
Sounds good!